app/js/lib/foxbox/webpush.js

```diff
@@ -80,10 +80,10 @@ export default class WebPush {
       })
       .catch((error) => {
         if (Notification.permission === 'denied') {
-          throw 'Permission request was denied.';
+          throw new Error('Permission request was denied.');
         } else {
```

Comment:
I see, you think it should work like the `no-else-return`. I don't think there is a specific rule for that. So maybe we should open an issue on eslint or write and contribute our own rule.